e2e/tests/init_test.go

```diff
@@ -243,15 +254,30 @@ func (s *suite) TheSessionForApplicationAndServiceContainsTheSupplier(appName st
 }
 
 func (s *suite) TheApplicationSendsTheSupplierARequestForServiceWithData(appName, supplierName, serviceId, requestData string) {
-	res, err := s.pocketd.RunCurl("", serviceId, requestData)
+	// TODO_TECHDEBT(#179): Once relayminer and appgateserver are running in tilt,
+	// use their respective in-tilt hostnames and run E2E tests in tilt. This
+	// should match the on-chain advertised endpoint for the service with the
+	// given serviceId.
+	res, err := s.pocketd.RunCurl("http://localhost:42069", serviceId, requestData)
```

Comment:
Thoughts on moving `localhost` into a local const and moving the comment along with it?

That way we're decoupling the "constant test state" and test parameters.